core/domain/stats_services.py

```diff
@@ -40,6 +40,44 @@ def get_top_unresolved_answers_for_default_rule(exploration_id, state_name):
     }
 
 
+def get_exp_wise_unresolved_answers_count_for_default_rule(exp_ids):
+    """Gets answer counts per exploration for the answer groups for default
+    rule across all states for explorations with ids in exp_ids.
+
+    Returns:
+        A dict, keyed by the string '{exp_id}', whose
+        values are the number of unresolved answers that exploration has.
+    """
+    explorations = exp_services.get_multiple_explorations_by_id(exp_ids)
+    
+    # The variable `exploration_states_tuples` is a list of all
+    # (exp_id, state_name) tuples for the given exp_ids.
+    # E.g. - [
+    #   ('eid1', 'Introduction'),
+    #   ('eid1', 'End'),
+    #   ('eid2', 'Introduction'),
+    #   ('eid3', 'Introduction')
+    # ]
+    # when exp_ids = ['eid1', 'eid2', 'eid3'].
+    explorations_states_tuples = [
+        (exp_domain_object.id, state_key)
+        for exp_domain_object in explorations.values()
+        for state_key in exp_domain_object.states
+    ]
+    exploration_state_list = get_top_state_rule_answers_multi(
```

Comment:
Perhaps call this `exploration_states_answers_list`? Right now it doesn't really convey that it is a list of answers.

(Sorry, I know I suggested this name earlier, but I was slightly confused then.)